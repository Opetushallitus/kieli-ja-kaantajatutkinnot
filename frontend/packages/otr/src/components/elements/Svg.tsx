import { FC } from 'react';

export const Svg: FC<{
  className?: string;
  src: string;
  alt: string;
}> = ({ className, src, alt }) => {
  return <img className={className} src={src} alt={alt} />;
};
