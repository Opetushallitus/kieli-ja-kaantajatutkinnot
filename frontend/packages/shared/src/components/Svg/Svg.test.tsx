import renderer from 'react-test-renderer';

import { Svg } from './Svg';
import OPHLogoENHorizontal from '../../statics/svg/oph_logo_horiz_en.svg';

describe('Svg', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Svg
          className="foo"
          src={OPHLogoENHorizontal}
          alt="alternate text goes here"
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
