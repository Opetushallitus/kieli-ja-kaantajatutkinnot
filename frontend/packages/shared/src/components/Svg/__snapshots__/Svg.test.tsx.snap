// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg should render correctly 1`] = `
<img
  alt="alternate text goes here"
  className="foo"
  src="test"
/>
`;
